controller/control.go

```diff
@@ -584,7 +584,7 @@ func (c *Controller) PrepareRestore(lastRestored string) error {
 	if c.isRestoring {
 		return fmt.Errorf("volume %v is restoring", c.Name)
 	}
-	if c.lastRestored != "" && c.lastRestored != lastRestored {
+	if c.lastRestored != "" && c.lastRestored != lastRestored && lastRestored != "" {
```

Comment:
OK, I cannot understand this commit without looking at the function using it. Merge it with the commit that using it.